RestInterface.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+#pylint : disable=E0401
+"""
+@author: PRATEEK CHANDRA	
+DATACORE SOFTWARE PVT LTD CONFIDENTIAL
+THIS SPEC IS THE PROPERTY OF DATACORE SOFTWARE PVT LTD.IT SHALL NOT BE
+COPIED, USED,TRANSLATED OR TRANSFERRED IN WHOLE OR IN PART TO ANY THIRD
+PARTY WITHOUT PRIOR WRITTEN PERMISSION OF DATACORE SOFTWARE PVT LTD.
+File Name	:	RestInterface.py
+Description	:	This script used to access SSY Rest API
+
+"""
+import json
+import requests
+BASE_URL = 'http://localhost/RestService/rest.svc/1.0/'
+USERNAME = "DcsAdmin"
+PASSWORD = "Datacore123"
```

Comment:
Is it good idea to add password in your script ?

Reply:
i will add it to config file milind